WordPress/Classes/Models/WPAccount+RestApi.swift

```diff
@@ -10,48 +10,48 @@ extension WPAccount {
     /// This used to be defined in the Objective-C layer, but we moved it here in a Swift extension in an attempt to decouple the model code from it.
     /// This was done in the context of https://github.com/wordpress-mobile/WordPress-iOS/pull/24165 .
     @objc var wordPressComRestApi: WordPressComRestApi? {
```

Comment:
I wasn't sure we can rely on overriding a property in an extension, so I'm now relying on a stored property in `WPAccount`.